client/src/pages/Homepage/Homepage.js

```diff
@@ -4,30 +4,27 @@ import './Homepage.css';
 class Homepage extends Component {
   render() {
     return (
-      <div id="content" className="container-fluid">
+      <div id="content">
         <div className="row">
```

Comment:
Don't think this class is necessary. Noticed that it's causing the page to use a horizontal scroll bar